ceph.spec.in

```diff
@@ -1719,8 +1730,14 @@ if test $STATUS -eq 0; then
     /usr/bin/systemctl stop ceph.target > /dev/null 2>&1
 fi
 
-# Now, relabel the files
-/usr/sbin/fixfiles -C ${FILE_CONTEXT}.pre restore 2> /dev/null
+# Relabel the files
+# Use ceph-disk fix for first package install and fixfiles otherwise
+if [ "$1" = "1" ]; then
+    /usr/sbin/ceph-disk fix --selinux
```

Comment:
@b-ranto it doesn't look like ceph-disk does too much to make this happen, would it be possible to just run the same shell command here instead? Or what was the difficulty you saw with that approach?